roles/confluent.common/tasks/main.yml

```diff
@@ -1,9 +1,4 @@
 ---
-- name: Confirm Hash Merging Enabled
```

Comment:
This is currently a "hard" requirement for using cp-ansible, and we are working on eliminating it. I would suggest to keep this check here.

Reply:
check is moved to the new task file, not removed